specification/dimensions/tags.md

```diff
@@ -0,0 +1,64 @@
+# Tags
+
+Tags are the set of finalized User-Defined and/or Provider-Defined Tags assigned to Tag Sources.  Tags commonly add business context to billing data to identify and accurately allocate charges.
+
+If a Tag is subject to a set of prioritized rules for determining its value, it is finalized after a single value is chosen from a set of corresponding values.  Otherwise, a Tag and a finalized Tag are equal.
+
+Tags adhere to the following requirements:
+
+* Tags MUST only contain finalized tags.
+* Tags MUST standardize to Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter User-Defined Tag keys.
+
+Provider-defined Tags additionally adhere to the following requirements:
+
+* Provider-defined Tags MUST be prefixed with a provider-defined prefix.
+* Providers SHOULD publish all provider-defined prefixes within their respective documentation.
+
+## Provider-Defined vs. User-Defined Tags
+
+The following is an example of one User-Defined Tag and one Provider-Defined Tag, respectively, with tag key, `foo`.  The first property is not prefixed, and the Provider has a pre-selected prefix, `marketplace/`, applied to denote that it is a different type of tag.
+
+```json
+    {
+        "foo":"bar",
+        "marketplace/foo": "bar"
+    }
+```
+
+## Finalized Tags
+
+A Tag can either be static or dynamic. If a Tag is static, its value is immutable. If a Tag is dynamic, its value is determined by a set of predefined user or provider rules. A finalized Tag is the final result of any static or dynamic Tag.
```

Comment:
I think we need to figure out if we need to keep capitalizing terms we are putting in the glossary. Maybe the first one gets capitalized and hyperlinked? And subsequent ones are just in lower case? This is a spec-wide decision but maybe clearest here because of the column and concepts Tag, Tags etc.

Reply:
+1 Capitalized and hyperlinked then... then column camelcased would be helpful to pick it out.  I have seen other columns defined like this.

Reply:
Everything capitalized is meant to be part of the Glossary

Reply:
@flanakin With your latest comment in mind then it'd be the following?

```suggestion
A tag can either be static or dynamic. If a tag is static, its value is immutable. If a tag is dynamic, its value is determined by a set of predefined user or provider rules. A finalized tag is the final result of any static or dynamic Tag.
```

vs. when we state the Tag column (above) we use Tag.